source/common/http/header_utility.cc

```diff
@@ -183,6 +183,19 @@ bool HeaderUtility::headerValueIsValid(const absl::string_view header_value) {
                                     header_value.size()) != 0;
 }
 
+bool HeaderUtility::headerValueContainsIgnoreCase(const absl::string_view header_value,
+                                                  const absl::string_view value,
+                                                  absl::string_view delimiter) {
+  std::vector<absl::string_view> sub_values =
+      StringUtil::splitToken(header_value, delimiter, true, true);
+  for (auto& sub_val : sub_values) {
+    if (absl::EqualsIgnoreCase(sub_val, value)) {
```

Comment:
I think this code won't match correctly if the original request has spaces after ',' in the transfer encoding string.

"transfer-encoding: identity,chunked" will match chunked, but "transfer-encoding: identity, chunked" won't match chunked.